src/Controls/ResourceGrid.cs

```diff
@@ -119,12 +122,25 @@ private void ApplyConditionalFormatting(DataGridViewRow r)
             {
                 ApplyConditionalCellFormatting(r.Cells[lng.LanguageId], SearchParams.TargetType.Text);
             }
+
+
+            if (!string.IsNullOrEmpty(r.Cells[colNameKey].Value?.ToString()) && r.Cells[colNameKey].Value.ToString().ToLower().Contains(".name"))

```

Comment:
Where does the `.name` come from and what does it mean? It should be explained in the readme at least.

Reply:
Working with forms localization and devexpress for example, it creates [control].name resx string. This usually is a control.name and change it is not a good idea